scripts/storage-emulator-integration/tests.ts

```diff
@@ -1254,25 +1234,31 @@ describe("Storage emulator", () => {
         AUTH_EMULATOR_HOST,
         STORAGE_EMULATOR_HOST.replace(/^(https?:|)\/\//, "")
       );
+    });
 
+    beforeEach(async function (this) {
+      this.timeout(TEST_SETUP_TIMEOUT);
       await resetEmulatorState();
-
       await testBucket.upload(image_filename, { destination: filename });
     });
 
-    afterEach(async () => {
-      await page.close();
+    afterEach(async function (this) {
+      this.timeout(EMULATORS_SHUTDOWN_DELAY_MS);
+      await page.evaluate(async () => {
+        await firebase.auth().signOut();
+      });
     });
 
-    after(async () => {
+    after(async function (this) {
+      this.timeout(EMULATORS_SHUTDOWN_DELAY_MS);
```

Comment:
We set this in line 104, I think we can remove this

Reply:
Unless we want the same timeout applied to Firebase Endpoints and the entire test suite?

Reply:
Moving to other PR